core/src/main/java/org/infinispan/remoting/responses/ClusteredGetResponseValidityFilter.java

```diff
@@ -33,7 +33,7 @@ public ClusteredGetResponseValidityFilter(Collection<Address> targets, Address s
    public boolean isAcceptable(Response response, Address address) {
       if (targets.contains(address)) {
          missingResponses--;
-         if (response instanceof SuccessfulResponse) {
+         if (response instanceof SuccessfulResponse || response instanceof ExceptionResponse) {
             validResponses++;
```

Comment:
`ExceptionResponse` is not a `ValidResponse`, so the `validResponses` field name is no longer accurate.